src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -467,35 +486,37 @@
          [react/view (style/message-view-wrapper outgoing)
           [message-timestamp message show-timestamp?]
           [react/view {:style (style/message-view message)}
-           [react/view {:style      (style/message-view-content)
-                        :max-height max-height}
+           [react/view {:style (style/message-view-content) :max-height max-height}
             (let [response-to (:response-to content)]
               [react/view {:on-layout
-                           #(when (and (> (.-nativeEvent.layout.height ^js %) max-message-height-px)
-                                       (not @collapsible?)
-                                       (not outgoing)
-                                       (not modal))
+                           #(when
+                             (and (> (.-nativeEvent.layout.height ^js %) max-message-height-px)
+                                  (not @collapsible?)
+                                  (not outgoing)
+                                  (not modal))
```

Comment:
unnecessary change

Reply:
My IDE is doing unnecessary changes to clojurescript formatting. Can you please tell me what IDE are you using?

Reply:
are you running `make lint-fix` before commiting?

I'm using vscode but not sure if it's the best option for the tech stack.

Reply:
Yes I am running `make lint-fix`. To fix all the unnecessary formatting changes caused by the IDE, I opened a new [PR](https://github.com/status-im/status-mobile/pull/13808). Please follow on the new [PR](https://github.com/status-im/status-mobile/pull/13808)